docs/apm/troubleshooting.asciidoc

```diff
@@ -95,13 +95,13 @@ Instead, we should strip away the unique information and group our transactions
 In this case, that means naming all blog transactions, `/blog`, and all documentation transactions, `/guide`.
 
 If you feel like you'd be losing valuable information by following this naming convention, don't fret!
-You can always add additional metadata to your transactions using {apm-guide-ref-v}/metadata.html#labels-fields[labels] (indexed) or
-{apm-guide-ref-v}/metadata.html#custom-fields[custom context] (non-indexed).
+You can always add additional metadata to your transactions using {apm-guide-ref}/metadata.html#labels-fields[labels] (indexed) or
+{apm-guide-ref}/metadata.html#custom-fields[custom context] (non-indexed).
 
 After ensuring you've correctly named your transactions,
-you might still see an error in the APM app related to too many transaction names.
-If this is the case, you can increase the default number of transaction groups displayed in the APM app by configuring
-<<apm-settings-kb,`xpack.apm.ui.transactionGroupBucketSize`>>.
+you might still see errors in the APM app related to transaction group limit reached (`_other`) or too many transaction names.
```

Comment:
I've updated the troubleshooting docs for transactions.
What do you think @sqren, @bmorelli25 ?

Reply:
The troubleshooting doc is already updated